recipes/mmft/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.1" %}
+{% set name = "mmft" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ARU-life-sciences/mmft/releases/download/v{{ version }}/source.tar.gz
+  sha256: eefe7e646f79b4cb09138495cac65d9fb0fa23c177eb131ed6984f41f6b78fe4
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('mmft', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+
+test:
+  commands:
+    - mmft --help
+    - mmft sample --help
+
+about:
+  home: https://github.com/ARU-life-sciences/mmft
+  license: MIT
+  summary: Max's minimal fasta toolkit
+  license_file: LICENSE
```

Comment:
Please see the Rust package guidelines for how to include the license of dependencies for Rust packages

https://bioconda.github.io/contributor/guidelines.html#rust